src/components/BackToTop.vue

```diff
@@ -13,37 +13,29 @@
   </div>
 </template>
 
-<script>
-
+<script setup>
 import { ref } from 'vue';
 
+const show = ref(false)
+
+let props = defineProps({targetId: {
```

Comment:
Prefer `const` since props are never reassigned
```suggestion
const props = defineProps({targetId: {
```